src/Vm.sol

```diff
@@ -188,156 +249,135 @@ interface VmSafe {
     // logs were emitted in the expected order with the expected topics and data (as specified by the booleans)
-    function expectEmit(bool, bool, bool, bool) external;
-    function expectEmit(bool, bool, bool, bool, address) external;
+    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData) external;
+    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData, address emitter)
+        external;
     // Mocks a call to an address, returning specified data.
     // Calldata can either be strict or a partial match, e.g. if you only
     // pass a Solidity selector to the expected calldata, then the entire Solidity
     // function will be mocked.
-    function mockCall(address, bytes calldata, bytes calldata) external;
+    function mockCall(address callee, bytes calldata data, bytes calldata returnData) external;
     // Mocks a call to an address with a specific msg.value, returning specified data.
     // Calldata match takes precedence over msg.value in case of ambiguity.
-    function mockCall(address, uint256, bytes calldata, bytes calldata) external;
+    function mockCall(address callee, uint256 msgValue, bytes calldata data, bytes calldata returnData) external;
     // Clears all mocked calls
     function clearMockedCalls() external;
     // Expects a call to an address with the specified calldata.
     // Calldata can either be a strict or a partial match
-    function expectCall(address, bytes calldata) external;
+    function expectCall(address callee, bytes calldata data) external;
     // Expects a call to an address with the specified msg.value and calldata
-    function expectCall(address, uint256, bytes calldata) external;
-    // Sets block.coinbase (who)
-    function coinbase(address) external;
+    function expectCall(address callee, uint256 msgValue, bytes calldata data) external;
+    // Sets block.coinbase
+    function coinbase(address newCoinbase) external;
     // Snapshot the current state of the evm.
     // Returns the id of the snapshot that was created.
     // To revert a snapshot use `revertTo`
-    function snapshot() external returns (uint256);
-    // Revert the state of the evm to a previous snapshot
+    function snapshot() external returns (uint256 snapshotId);
+    // Revert the state of the EVM to a previous snapshot
     // Takes the snapshot id to revert to.
     // This deletes the snapshot and all snapshots taken after the given snapshot id.
-    function revertTo(uint256) external returns (bool);
+    function revertTo(uint256 snapshotId) external returns (bool success);
     // Creates a new fork with the given endpoint and block and returns the identifier of the fork
-    function createFork(string calldata, uint256) external returns (uint256);
+    function createFork(string calldata endpoint, uint256 blockNumber) external returns (uint256 forkId);
```

Comment:
```suggestion
    function createFork(string calldata url, uint256 blockNumber) external returns (uint256 forkId);
```

Reply:
`urlOrAlias` is probably best as @paulrberg pointed out above, since both are supported